Packs/OctoxLabs/README.md

```diff
@@ -0,0 +1,166 @@
+## Configure OctoxLabs on Cortex XSOAR
```

Comment:
Let's switch the text in this file:
Add few sentences about the product it self
Also, add some information about what this pack do.
You can take `Packs/PAN-OS/README.md` as a good reference.

Reply:
Thank you for your review. I updated it.
[https://github.com/demisto/content/pull/21175/commits/bb65dacdb0fa299b850e2cf0be62d456d3ada744](https://github.com/demisto/content/pull/21175/commits/bb65dacdb0fa299b850e2cf0be62d456d3ada744)